### DISCUSSION TITLE
Where is the Home Assistant Forum Thread?

### DISCUSSION THREAD

**Discussion by RichardUUU** [mikejac/node-red-contrib-google-smarthome#449] [General]
Is there a main thread on the Home Assistant forum to discuss this integration?

**Comment by Paul-Reed** [comment]
Have you asked Home Assistant?

**Comment by Caprico85** [comment]
We have nothing to do with Home Assistant. We are (or at least I am) not active on the Home Assistant Forum, therefore we have no "official" threads over there.
